frontend/src/components/common/Footer.tsx

```diff
@@ -0,0 +1,99 @@
+import {
+  Container,
+  Flex,
+  Link,
+  Text,
+  useMediaQuery,
+  VStack,
+} from "@chakra-ui/react";
+import React from "react";
+
+import { Facebook, Instagram } from "./icons";
+
+const Footer = (): JSX.Element => {
+  const [isScreenLargerThan] = useMediaQuery("(min-width: 320px)");
```

Comment:
```suggestion
```
we don't need to use a media query, can pass in responsive styles to the container instead: 
i.e. `mt={{ base: 4, md: 0 }}`

this way it'll be more reusable in the future if we want to support more than 2 sizes (removes the ternary)